Logging/src/LogMessageProcessor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Logging;
+
+/**
+ * Interface representing a Message Processor used by the {@see PsrLogger}
+ */
+interface LogMessageProcessor
```

Comment:
We've trended towards suffixing interfaces with `Interface`, would you be open to doing the same here?

Reply:
Sure!

Really just out of curiosity, I surely don't want to start a flame war 😅: what made you trend towards it? (The old "`interface x` and `y implements x` already tells that it is an interface" argument)

Reply:
I've found it nice while browsing the file tree to be able to quickly identify things like traits/interfaces vs. classes.